attest/core/rustfmt.toml

```diff
@@ -0,0 +1,7 @@
+edition = "2018"
```

Comment:
Should we put this stuff in the root rustfmt.toml and just reformat everything in a separate PR?